plugins/profefe.yaml

```diff
@@ -3,7 +3,7 @@ kind: Plugin
 metadata:
   name: profefe
 spec:
-  version: v0.9.0
+  version: v0.3.1
```

Comment:
@ahmetb I think this was wrong since day 1 :) luckily now I am using the released @rajatjindal made.
So it should be all right!!

Reply:
This is trouble, look here https://github.com/kubernetes-sigs/krew/blob/ba340341aeec30ec99bbe3c547dfc5ba2eeea395/internal/installation/upgrade.go#L62

So nobody who already installed your plugin will upgrade to a new version until you release v0.9.1. Would that be an option for you?

/hold

Reply:
Yep sadly, this will not ship the new version to the users, until it's called v0.9.1.

Temporarily, you can call it `v0.9.1+but-actually-v0.3.1` until your version number catches up, but that's quite silly :)

Reply:
I think it is time to bump to v0.10 !!